### PR TITLE
Fix link of pydocstyle in contributing

### PR DESCRIPTION
**Pull request by ferhatelmas**
## Description of proposed changes

Correct pydocstyle link in contributing.

## Related issue(s)

N/A

## Test plan

N/A

## Checklist

Need help on these? Just ask!

* [X] I have read the **CONTRIBUTING** document.
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.
* [ ] All new and existing tests passed.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ We use the following packages:
 * [black](https://black.readthedocs.io/en/stable/): automatic code formatting
 * [flake8](http://flake8.pycqa.org/en/latest/): PEP8 linting
 * [mypy](http://mypy-lang.org/): static type checking
-* [pydocstyle](http://mypy-lang.org/): docstring compliance
+* [pydocstyle](https://readthedocs.org/projects/pydocstyle/): docstring compliance
```
[the remark this answers is not in the document]
Comment (ferhatelmas) [reply CONTRIBUTING.md]: Updated.

**Comment by henryre** [comment]
@ferhatelmas going to go ahead and merge myself this since it's a small change. thanks again!

**Comment by ferhatelmas** [comment]
@henryre thank you. If you point me in the right direction, I want to contribute more.

**Comment by henryre** [comment]
@ferhatelmas that's great to hear, always looking forward to more contributions! Our help-wanted issues are a great place to start: https://github.com/snorkel-team/snorkel/labels/help%20wanted. If you want to tackle one, post a plan in the Issue and tag a maintainer to discuss.

**Comment by ferhatelmas** [comment]
Thank you, noted. I will start with them and will let the maintainers if I take something seriously or have any questions.

**Pull request merged**